ios/RNRadar.m

```diff
@@ -1152,9 +1152,9 @@ - (void)didUpdateToken:(RadarVerifiedLocationToken *)token  {
     };
     
     if (revenue) {
-        [Radar logConversionWithName:name metadata:metadata completionHandler:completionHandler];
-    } else {
         [Radar logConversionWithName:name revenue:revenue metadata:metadata completionHandler:completionHandler];
+    } else {
```

Comment:
When we're parsing revenue, let's do this:
```
    NSNumber *revenue = nil;
    if (revenueObj != nil) {
        if ([revenueObj isKindOfClass:[NSNumber class]]) {
            revenue = (NSNumber *)revenueObj;
        } else if ([revenueObj isKindOfClass:[NSString class]]) {
            revenue = @([(NSString *)revenueObj doubleValue]);
        }
    }
  ```

and then change `if (revenue) {` to `if (revenue != nil) {`

Reply:
we don't need to handle the case where it is a string as the ts is types as number.

Reply:
<img width="579" alt="image" src="https://github.com/user-attachments/assets/dec5e087-29fb-42e4-a2c5-07508a633703">
<img width="341" alt="image" src="https://github.com/user-attachments/assets/233b3a5d-8bba-4b54-927d-21002188b1c6">

Dogfooding to ensure that the current implementation handles having revenue defined as param or missing in param